src/plugin/cesium/cesiumrenderer.js

```diff
@@ -377,7 +377,7 @@ plugin.cesium.CesiumRenderer.prototype.toggleMovement = function(value) {
   if (this.olCesium_) {
     var scene = this.olCesium_.getCesiumScene();
     if (scene && scene.screenSpaceCameraController) {
-      scene.screenSpaceCameraController.enableInputs = value;
+      scene.screenSpaceCameraController.enableInputs = !!value;
```

Comment:
`value` is typed as `boolean`, so this seems to be a superfluous addition. It looks like the missing param in my previous comment is causing the compiler to think the type is `boolean|undefined`.